Web_Development/FrontEnd/High Resolution Display in CSS/Retina Display for High Resolution.md

```diff
@@ -0,0 +1,47 @@
+# High Resolution Display using the Retina Image Concept
+
+Apple introduced a display called the "Retina Display" that has much better resolution than our normal display, such as a picture element per inch (PPI) or dots per inch (DPI). This display has high resolution power, and we can easily distinguish between normal and retina displays because each display varies in size and specifications.
+
+## Display Resolution
+
+The display resolution of the device, like a laptop monitor or smart TV, has a variety of different pixels in every space that may be displayed. In digital mensuration, the display resolution is measured in pixels per inch (PPI).
+
+There are two types of display resolution we can define:-
+
+1) Low Display Resolution: The display on which pixels per inch have a low value is called low display resolution. It is less sharp as compared to high resolution.
+2) High Display Resolution: The display in which pixels per inch have a higher value as compared to others is called a "high display resolution" and it is more clear.
+
+## Retina Display
+
+A Retina display is a term created by Apple to describe devices and monitors that have a resolution and element density so high that someone is unable to recognise the individual pixels at a traditional viewing distance. Now, The Apple device's display was well received and greatly improved the viewing experience.
+
+Difference between Non Retina Display and Retina Display
+
+1. Retina Display Image
+
+![300px-Retina_Display](https://user-images.githubusercontent.com/54473091/142192529-16602e16-c339-44de-915d-ddc02687b69d.jpg)
+
+[Retina Display Image](https://user-images.githubusercontent.com/54473091/142192529-16602e16-c339-44de-915d-ddc02687b69d.jpg)
+
+2. Non Retina Display Image
+
+![300px-Non-Retina_Display](https://user-images.githubusercontent.com/54473091/142193055-64f38832-168c-45ed-94ee-d02bfb779dbb.jpg)
+
+[Non Retina Display Image](https://user-images.githubusercontent.com/54473091/142193055-64f38832-168c-45ed-94ee-d02bfb779dbb.jpg)
```

Comment:
Same here,
Place a link to the actual source.
AFAIK you've taken the images from [here](https://en.wikipedia.org/wiki/Retina_display).

Do update the captions to real links to source.